deps/rabbit/src/rabbit_definitions.erl

```diff
@@ -464,6 +464,12 @@ add_policy(Param, Username) ->
 
 add_policy(VHost, Param, Username) ->
     Key   = maps:get(name,  Param, undefined),
+    case Key == undefined of
+      true ->
+        rabbit_misc:protocol_error(
```

Comment:
While this ends up calling `erlang:exit/1` and technically might be how other errors are handled in this function,
`rabbit_misc:protocol_error/3` is mean to be used to terminate a channel and return an AMQP 0-9-1 error back to the client. It's therefore not appropriate here.

Reply:
Please use `exit/1` directly as the same function does if `rabbit_policy:set/7` returns an error.